benchmark/distributed/worker.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const yargs = require('yargs');
+
+const jstp = require('../..');
+const parseAddress = require('./parse-address');
+
+const args = yargs
+  .option('address', {
+    alias: 'a',
+    describe: 'Address of master server',
+    type: 'string',
+    default: 'tcp://localhost:3000',
+  })
+  .option('connection', {
+    alias: 'c',
+    describe: 'Amount of connections',
+    type: 'number',
+    default: 128,
+  })
+  .option('interval', {
+    alias: 'i',
+    describe: 'Interval of requests sent by connection in ms',
+    type: 'number',
+    default: 10,
+  })
+  .option('size', {
+    alias: 's',
+    describe: 'Size of argument to call serverside method with',
+    type: 'number',
+    default: 0,
+  })
+  .option('report', {
+    alias: 'r',
+    describe: 'Report interval in ms',
+    type: 'number',
+    default: 10000,
+  })
+  .help()
+  .alias('help', 'h')
+  .argv;
+
+const {
+  address,
+  size: argumentSize,
+  connection: connectionAmount,
+  interval: requestInterval,
+  report: reportInterval,
+} = args;
+
+const { transport, connect: masterAddress } = parseAddress(address);
+
+// Prevent Node.js from throwing DEPTH_ZERO_SELF_SIGNED_CERT.
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const connections = new Array(connectionAmount);
+const timers = new Array(connectionAmount);
+
+const argument = '0'.repeat(argumentSize);
+
+let requests = 0;
+
+const createConnection = (index, address) => {
+  jstp[transport].connectAndInspect(
+    'server', null, ['server'], ...address, (error, connection, api) => {
+      if (error) {
+        console.error(`Could not connect to server:\n${error}`);
+        return;
+      }
+
+      connections[index] = connection;
+      timers[index] = setInterval(() => {
```

Comment:
I think it's better to set one interval timer and make all calls in there, this will most likely make clients more performant.